metpy/calc/basic.py

```diff
@@ -257,6 +257,77 @@ def pressure_to_height_std(pressure):
     return (t0 / gamma) * (1 - (pressure / p0).to('dimensionless')**(Rd * gamma / g))
 
 
+@exporter.export
+@check_units('[length]')
+def height_to_geopotential(height):
+    r"""Computes geopotential for a given height in meters
+
+    Parameters
+    ----------
+    height : `pint.Quantity`
+        Height above sea level (array_like)
+
+    Returns
+    -------
+    `pint.Quantity`
+        The corresponding geopotential value(s)
+
+    Examples
+    --------
+    >>> from metpy.constants import g,Re,G,me
+    >>> import metpy.calc
+    >>> from metpy.units import units
+    >>> height = np.linspace(0,10000, num = 11) * units.m
+    >>> geopot = metpy.calc.height_to_geopotential(height)
+    >>> geopot
+    <Quantity([     0.           9817.70342881  19632.32592389  29443.86893527
+    39252.33391207  49057.72230251  58860.0355539   68659.27511263
+    78455.4424242   88248.53893319  98038.56608327],
+    'meter ** 2 / second ** 2')>
+    """
+    # Calculate geopotential
+    geopot = G * me * ((1 / Re) - (1 / (Re + height)))
+
+    return geopot
+
+
+@exporter.export
+def geopotential_to_height(geopot):
+    r"""Computes height from a given geopotential
+
+    Parameters
+    ----------
+    geopotential : `pint.Quantity`
+        Geopotential (array_like)
+
+    Returns
+    -------
+    `pint.Quantity`
+        The corresponding height value(s)
+
+    Examples
+    --------
+    >>> from metpy.constants import g,Re,G,me
+    >>> import metpy.calc
+    >>> from metpy.units import units
+    >>> height = np.linspace(0,10000, num = 11) * units.m
+    >>> geopot = metpy.calc.height_to_geopotential(height)
+    >>> geopot
+    <Quantity([     0.     9817.70342881  19632.32592389  29443.86893527
+    39252.33391207  49057.72230251  58860.0355539   68659.27511263
+    78455.4424242   88248.53893319  98038.56608327],
+    'meter ** 2 / second ** 2')>
+    >>> height = metpy.calc.geopotential_to_height(geopot)
+    >>> height
+    <Quantity([     0.   1000.   2000.   3000.   4000.   5000.   6000.   7000.
+    8000. 9000.10000.], 'meter')>
```

Comment:
Need some more separation spaces, especially between 9000. and 10000. So should be:
```
8000.  9000.  10000.
```